server.ts

```diff
@@ -30,6 +32,7 @@ const oktaIssuerUrl = <string>process.env.OKTA_ISSUER_URL;
 const appBaseUrl = <string>process.env.APP_BASE_URL;
 
 const conferenceCodeLength = 3;
+const participantLifetimeMillis = 9 * 60 * 60 * 1000; // 9 hours
```

Comment:
It might be cool for the platform to return how long a participant JWT is good for to enable a more dynamic version of the backup participant cleanup I implemented here.

Reply:
I like that idea, I think it is always good to be able to query the source for stuff like this 👍